<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>load_chat</class>
 <widget class="QDialog" name="load_chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Chat</string>
  </property>
  <widget class="QPushButton" name="btn_select_config">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select config</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pte_user_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tb_chat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>541</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_enter">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>660</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_config_path">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected config file path</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
